apify-docs-theme/src/theme/custom.css

```diff
@@ -1560,7 +1560,6 @@ html[data-theme='dark'] .runnable-code-block svg .apify-logo {
     min-width: 1.5rem;
     display: inline-block;
     opacity: .3;
-    position: sticky;
```

Comment:
Now it's the same as on web, the numbers just scroll with the code (e.g. here https://apify.com/compass/google-maps-extractor/api/client/nodejs )